ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -32,6 +32,11 @@ cat > packet-setup.yaml <<-EOF
       msg: "Unsupported CLUSTER_TYPE '{{ cluster_type }}'"
     when: "cluster_type is not regex('^packet.*$|^equinix.*$')"
 
+  - name: check deprecated cluster type
+    fail:
+      msg: "Deprecated CLUSTER_TYPE '{{ cluster_type }}'"
+    when: cluster_type == "packet"
+
```

Comment:
I think it could be useful to have the same check in the teardown step as well (since it gets invoked always).
Also maybe this check could be done before the `check cluster type`

Reply:
Thanks @andfasano. These are all valid points and changed the PR that way.